index-files/index-1.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
What's the deal with the index-*.html files? If it's an index file it should be index.html, and if it's a specific source file it should be named in such a way to suggest what kind of content it actually has